source/utilities.tex

```diff
@@ -12349,7 +12349,7 @@
 
 \indexlibraryctor{default_searcher}%
 \begin{itemdecl}
-constexpr default_searcher(ForwardIterator pat_first, ForwardIterator pat_last,
+constexpr default_searcher(ForwardIterator1 pat_first, ForwardIterator1 pat_last,
```

Comment:
Should we change the synopsis to match?

Reply:
It's correct in the synopsis. (ForwardIterator1 is a template parameter of the enclosing class.)

Reply:
Right, but in the synopsis it's just called `ForwardIterator`. It's not incorrect, just inconsistent.

Reply:
You were talking about the header synopsis; I was looking at the class synopsis (and found not issue).

The header synopsis is now fixed.

Reply:
Yes, sorry for being unclear! I should have said header.